packages/amplify-e2e-core/src/utils/credentials-rotator.ts

```diff
@@ -54,16 +64,24 @@ export const tryScheduleCredentialRefresh = () => {
     return;
   }
 
-  if (!process.env.USE_PARENT_ACCOUNT) {
-    throw new Error('Credentials rotator supports only tests running in parent account at this time');
-  }
+  if (process.env.USE_PARENT_ACCOUNT) {
+    // Attempts to refresh credentials in background every 15 minutes.
```

Comment:
Suggestion.

I'd reduce interval to something smaller (5 minutes)?

The reason is - test framework spawns subprocess that inherit env vars. So their credential validity period is `how_much_time_since_last_rotation + subprocess_lifespan`. Some of these processes take long time.

Reply:
In searchable tests (in cli repo) this refresh was (is?) too sparse sometimes.  (we haven't adjusted this on CLI side though).